bindings/python/src/operator.rs

```diff
@@ -179,8 +179,8 @@ impl Operator {
     /// # Notes
     ///
     /// - Delete not existing error won't return errors.
-    pub fn delete(&self, path: &str) -> PyResult<()> {
-        self.core.delete(path).map_err(format_pyerr)
+    pub fn exists(&self, path: &str) -> PyResult<(bool)> {
```

Comment:
Hi, thank you for your contribution. However, we need to add a new API instead of removing the old one. Please double check the original issues.